packages/applet/src/modules/components/index.vue

```diff
@@ -37,6 +37,7 @@ const [filtered, toggleFiltered] = useToggle(true)
 const componentTreeLoaded = ref(false)
 const inspectComponentTipVisible = ref(false)
 const componentRenderCode = ref('')
+const showComponentRenderCode = ref(false)
```

Comment:
Let's change to `componentRenderCodeVisible`

Reply:
Pushed